commands/src/main/java/org/jclouds/karaf/commands/table/internal/OSGiScriptEngine.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
> Do you plan to re-style the classes to align them to our coding style and change them to cleanup the code? 

I think we should certainly align the code style; not sure about more comprehensive re-writes, though. Agree that we should change the functionality (esp. the silent failures) where we feel we should behave differently.